app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
+    before_filter :validate_item, only: [:create, :update]
```

Comment:
Mike, I had to disable `protect_from_forgery` in the demo app's (spec/instance) ApplicationController -- is this enabled by default in Sufia? If so, maybe we could add `skip_before_action :verify_authenticity_token` for JSON requests to `[:create, :update, :destroy]`.